.github/workflows/coverage.yml

```diff
@@ -10,30 +11,81 @@ on:
         required: false
         type: string
         default: npx jest --coverage
+      comparison_branch:
+        required: false
+        type: string
+        default: main
     secrets:
       NPM_TOKEN:
         required: false
       caller_github_token:
         required: true
 
 jobs:
-  Coveralls:
+  base-coverage:
+    runs-on: ubuntu-latest
+    outputs:
+      coverage-percent: ${{ steps.parse-coverage.outputs.coverage-percent }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.comparison_branch }}
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm ci --ignore-scripts
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - run: npm rebuild
+      - run: ${{ inputs.test_script }}
+      - uses: yext/slapshot-reusable-workflows/get-coverage-percent@dev/coverage-workflow-prescan
```

Comment:
will the branch be updated after merging into main?

Reply:
just saw that you updated!

Reply:
it's updated already!